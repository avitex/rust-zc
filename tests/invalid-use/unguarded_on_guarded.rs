use zc::Guarded;

#[derive(Guarded)]
#[zc(unguarded)]
pub struct StructWithDefault<'a>(&'a u8);

fn main() {}
