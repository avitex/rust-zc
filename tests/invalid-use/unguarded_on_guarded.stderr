error: attempting to disable `zc::Guarded` implementation while deriving `zc::Guarded`
 --> $DIR/unguarded_on_guarded.rs:4:1
  |
4 | #[zc(unguarded)]
  | ^
